README.md

```diff
@@ -1,2 +1,63 @@
 # Terraform Provider Xray
-[Xray API Documentation](https://www.jfrog.com/confluence/display/JFROG/Xray+REST+API)
\ No newline at end of file
+
+[![Actions Status](https://github.com/jfrog/terraform-provider-xray/workflows/release/badge.svg)](https://github.com/jfrog/terraform-provider-xray/actions)
+[![Go Report Card](https://goreportcard.com/badge/github.com/jfrog/terraform-provider-xray)](https://goreportcard.com/report/github.com/jfrog/terraform-provider-xray)
+[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/jfrog/terraform)
+
+To use this provider in your Terraform module, follow the documentation [here](https://registry.terraform.io/providers/jfrog/xray/latest/docs).
+[Xray general information](https://jfrog.com/xray/)
+[Xray API Documentation](https://www.jfrog.com/confluence/display/JFROG/Xray+REST+API)
+
+## Release notes for v0.0.1
+Xray provider was separated from Artifactory provider. The most notable differences in the new Xray provider: 
+- Provider uses Xray API v2 for all the API calls.
+- HCL was changed and now uses singular names instead of the plurals for the repeatable elements, like `rule`, `watch_resource`, `filter` and `assigned_policy`.
+- Security policy and License policy now are separate Terraform provider resources.
+- In Schemas, TypeList was replaced by TypeSet (where it makes sense) to avoid sorting problems, when Terraform detect the change in sorted elements.
+- Added multiple validations for Schemas to verify the data on the Terraform level instead of getting errors in the API response.
+
```

Comment:
Please include the list of limitation of this release, e.g. not supporting JSON object in filter value.

Reply:
done